pkgs/servers/apache-directory-server/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchzip, adoptopenjdk-jre-openj9-bin-11, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "apache-directory-server";
+  version = "2.0.0.AM26";
+
+  src = fetchzip {
+    url = "https://dlcdn.apache.org//directory/apacheds/dist/${version}/apacheds-${version}.zip";
+    sha256 = "sha256-36kDvfSy5rt/3+nivEFTepnIKf6sX0NTgPRm28M+1v4=";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/java
+    install -D $src/lib/*.jar $out/share/java
+    CLASSPATH=$(JARS=($out/share/java/*.jar); IFS=:; echo "''${JARS[*]}")
```

Comment:
Is it likely that share/java will have a collision?

Reply:
Great catch! Looking at some other Java packages, I've noticed that they put things in `share/<name>`, so I've done the same.